src/test/java/uk/gov/hmcts/reform/divorce/orchestration/tasks/ccdcase/PatchCaseInCCDTest.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.divorce.orchestration.tasks.ccdcase;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.divorce.orchestration.client.CMSClient;
+import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.task.DefaultTaskContext;
+import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.task.TaskContext;
+
+import java.util.Collections;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.divorce.orchestration.TestConstants.AUTH_TOKEN;
+import static uk.gov.hmcts.reform.divorce.orchestration.TestConstants.TEST_CASE_ID;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.AUTH_TOKEN_JSON_KEY;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.CASE_ID_JSON_KEY;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PatchCaseInCCDTest {
+    private static final Map<String, Object> TEST_DATA = Collections.emptyMap();
+
+    @Mock
+    private CMSClient caseMaintenanceServiceClient;
+
+    @InjectMocks
+    private uk.gov.hmcts.reform.divorce.orchestration.tasks.ccdcase.PatchCaseInCCD patchCaseInCCD;
```

Comment:
can't see another import for `PatchCaseInCCD` ? so can this be imported on top ?